examples/StandAloneMatLab/testScript2.py

```diff
@@ -0,0 +1,13 @@
+import sys
+

```

Comment:
Where is this script used?

Reply:
This script is called with some of the example configs. It's purpose is to demonstrate the ability to pass multiple command line arguments with the matlab agent.